pyproject.toml

```
@@ -29,6 +29,7 @@ opencv-python = {version="4.2.0.32", optional=true}
 xlrd = {version="1.2.0", optional=true}
 freezegun = {version="0.3.15", optional=true}
 signalr-client-threads = {version="0.0.12", optional=true}
+tqdm = "^4.59.0"
```

As this is only used by the parsers, we should add the `optional` flag here. I also lowered the version here to support the version we are using internally.

```suggestion
tqdm = {version="^4.19.5", optional=true}
```

We should also add `tqdm` to the list to `parsers` in the `[tool.poetry.extras]` section.

---

> We should also add tqdm to the parsers list in the [tool.poetry.extras] section.

Reminder to update the pyproject.toml file.

---

@corradio

---

Let me know if https://github.com/tmrowco/electricitymap-contrib/pull/3066/commits/407f0117520b9381178042311a4942b733299efe fixes it